manager/assets/modext/util/utilities.js

```diff
@@ -347,18 +347,20 @@ Ext.override(Ext.form.Action.Submit,{
 
 /* QTips to form fields */
 Ext.form.Field.prototype.afterRender = Ext.form.Field.prototype.afterRender.createSequence(function() {
-    if (this.description) {
+    if (this.description && (MODx.config.manager_tooltip_enable === true)) {
```

Comment:
```suggestion
    if (this.description && MODx.config.manager_tooltip_enable) {
```

Reply:
I'm like 95% sure the check as it is now (`=== true`) is not going to work. The config object is not strictly typed, as settings are stored as text, so a strict type check for true will always be false. The value of `MODx.config.manager_tooltip_enable` will in most cases be a string `'1'` or `'0'`/`''`.